src/MessagePack.UnityClient/Assets/packages.config

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
```

Comment:
Users can resolve dependencies through NuGetForUnity,
but this project itself pulls dlls from bin/Debug,
so because dependencies can't be resolved, we're manually inserting them.

While it would be best if we could synchronize with the csproj,
this is only for running during development (necessary when running unit tests),
so I think it's probably okay if we're not too strict about versions.